CHANGES

```diff
@@ -2,6 +2,11 @@
 CHANGES
 =======
 
+Unreleased
```

Comment:
This should be the actually next release version. We don't do any dev/beta/snapshot releases.